AK/DOSPackedTime.h

```diff
@@ -31,11 +31,15 @@ union DOSPackedDate {
 };
 static_assert(sizeof(DOSPackedDate) == 2);
 
-inline constexpr u16 first_dos_year = 1980;
+inline constinit u16 first_dos_year = 1980;
+inline constinit u32 first_dos_representable_unix_timestamp = UnixDateTime::from_unix_time_parts(1980, 1, 1, 0, 0, 0, 0).seconds_since_epoch();
+inline constinit u64 last_dos_representable_unix_timestamp = UnixDateTime::from_unix_time_parts(2107, 12, 31, 23, 59, 59, 0).seconds_since_epoch();
```

Comment:
`constexpr` was correct here. `constinit` is weaker than `constexpr` and just means it needs a constant initializer. `constinit` doesn't define a compile-time constant.

(the names are a bit confusing)